### ISSUE TITLE
Pension List+Loop: Inconsistent behavior when “Doesn’t have a Social Security number” is checked

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#105393]
## Brief description
As part of our list and loop enhancements, we submitted an [ADE ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104338) for a11y review. This is one of the findings to investigate and address before we release the component live.

## Problem - Inconsistent behavior when “Doesn’t have a Social Security number” is checked

**Page**: Add a dependent child, [Social Security information](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/dependents/0/social-security-number?add=true) page
**Device**: MacOS
**Expected severity level**: 1
**Expected experience standard**: User can complete necessary flows
**Description**: In one instance, when the “Doesn’t have a Social Security number” checkbox is checked, and the user hits “Continue,” they are unable to progress to the next page. They get an error telling them to put in an SSN:

![Image](https://github.com/user-attachments/assets/e4c3643c-5820-4592-b0fb-b77f51b8434f)

![Image](https://github.com/user-attachments/assets/b4dc7f58-8b13-4378-aabd-48aa775d9b50)

**How to recreate:** 
1. Go to the Add a dependent child flow, SS information page
2. Select “Doesn’t have a social security number”
3. Select “Continue” - does it allow you to proceed?
4. Go back, enter an SSN, and select “Doesn’t have a social security number”
5. Select “Continue” - does it stop you from proceeding?

**Recommendation**: Determine whether SSNs are required or not. If they are, remove the checkbox. If they aren’t, make sure that the user can progress if the checkbox is checked.
